packages/jest-haste-map/src/index.ts

```diff
@@ -113,6 +113,9 @@ const CHANGE_INTERVAL = 30;
 const MAX_WAIT_TIME = 240000;
 const NODE_MODULES = path.sep + 'node_modules' + path.sep;
 const PACKAGE_JSON = path.sep + 'package.json';
+const VCS_DIRECTORIES = ['.git', '.svn', '.hg']
+  .map(vcs => '/' + vcs + '/')
```

Comment:
This should use ^ and $, right?

Reply:
should probably use `path.sep` like above as well?

Reply:
Good catch, changed to `path.sep`. 

I don't think I understand why would this need `^` and `$`? it matches with all the strings that include `path.sep + '.git' + path.sep` and would not match with `path.sep +'.github' + path.sep`.